sdk/cosmos/azure_data_cosmos/tests/cosmos_databases.rs

```diff
@@ -0,0 +1,116 @@
+mod framework;
+
+use std::error::Error;
+
+use azure_core_test::{recorded, TestContext};
+use azure_data_cosmos::{models::ThroughputProperties, CreateDatabaseOptions, Query};
+use framework::TestAccount;
+use futures::StreamExt;
+
+#[recorded::test(live)]
+#[cfg(feature = "key_auth")]
```

Comment:
Once we have tests recordings - my goal before I leave for the year - will we be able to pass `--all-features` to our CI tests? I.e., can the impact here be recorded (perhaps with sanitizers/matches)?

Reply:
The test can run under `--all-features` already. In fact, Cosmos is ready to roll with `--all-features` (and I would, in fact, prefer we enable it ASAP, to catch `key_auth`-related tests).

Right now, it's "live only" because we have no recordings, but it's should work fine under recorded tests.

I should note that in all our other language SDKs, Cosmos has per-PR test runs that use the Emulator (which act as "live" tests because they talk to a "real" emulator). We'll be looking to do the same here too.

Reply:
@hallipr we'll want to look at how other languages "extend" the CI pipeline and do the same here such that Cosmos (and probably Storage) can run emulators as well.

Reply:
The plan was always to follow suit with the other languages. @hallipr has some of that working already, but I don't think it fully ingests individual `ci.yml` files like in other languages yet.